source/content/redis.md

```diff
@@ -217,9 +217,6 @@ This configuration uses the `Redis_CacheCompressed` class for better performance
     $conf['cache_default_class'] = 'Redis_CacheCompressed';
     $conf['cache_prefix'] = array('default' => 'pantheon-redis');
 
-    // Do not use Redis for cache_form (no performance difference).
```

Comment:
For technical review: Move this change to the D8 tab, also from troubleshooting.